### PR TITLE
EN_Reboot: Various fixes

### PR DESCRIPTION
**Pull request by Frey8914**
Fix Block Move.
Fix dialogue.
Fix incapacitation.
Fix Lobby Actions.
Fix Login Bonus
Fix options.
Fix Regional Mags.
Fix Sub Missions.
Fix Tasks.
Fix tutorials.
Fix UI.
Fix Shops.
Fix Item Lab.
Fix Class Counter.
Fix Mining Rig Defense.
Fix Seasonal Events
Fix terms: Hounyan, Hybrid Instructor, leveling, catalyze, Main Menu, Normal Attack, Season Enemy
Fix misc: spelling, text code, spacing

Addendum:
Fix Character Info window.
Fix Lobby Action window.
Fix untranslated option.

### CODE REVIEW

File: Translated/Dialog/Deirdre/ssn_014270.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name_7400#0,"""Deirdre"""
 ssn_014270_0010#0,"""Hey, ain't seen you around here before. Newly arrived to Central City, are ya?"""
-ssn_014270_0020#0,"""You should come see me after you get some notches on your belt. Might be as I can teach you a thing or two."""
```
Comment (allister-translator) [inline Translated/Dialog/Deirdre/ssn_014270.csv]: i already told you this was intentional on my part. pls leave my fluff alone

Comment (Frey8914) [reply Translated/Dialog/Deirdre/ssn_014270.csv]: Whatever it is trying to convey is coming off as bad English.

Comment (DabirSA) [reply Translated/Dialog/Deirdre/ssn_014270.csv]: It's a perfectly fine colloquialism. Revert this.

File: Translated/Dialog/Deirdre/ssn_014271.csv [hunk 2]
```diff
@@ -4,7 +4,7 @@ ssn_014271_0101#0,"""...Mm? Doesn't look like your Class is Braver."""
 ssn_014271_0102#0,"""Gonna have to get you to sign up as a Braver at the Class Counter first. Gets the formalities out of the way, y'know?"""
 ssn_014271_0103#0,"""While you're there, give Braver's Class Skills a good looking-over. Commit 'em to memory."""
 ssn_014271_0104#0,"""But hey, I might as well tell you what the Class is all about while you're here."""
-ssn_014271_0201#0,"""Bravers are a Class geared toward excelling at both close- and long-ranged combat."""
```
Comment (allister-translator) [inline Translated/Dialog/Deirdre/ssn_014271.csv]: this is a hanging hyphen. those are a thing.

File: Translated/Dialog/Deirdre/ssn_014270.csv (hunk 1 above)
Comment (allister-translator) [reply Translated/Dialog/Deirdre/ssn_014270.csv]: deridre doesn't care if you think she uses bad english. she is a tomboy queen and i will convey her message 😤

File: Translated/Tasks/Side/Hoxie/ss0030761201.csv [hunk 3]
```diff
@@ -1,3 +1,3 @@
 Title#0,"""Aerio Subjugation Exercise III"""
-Summary#0,"""The gatekeeper of Central City's North Gate, Hoxie, recommends you defeat enemies at Resol Forest to gain more field experience."""
+Summary#0,"""The gatekeeper of Central City's Northern Gate, Hoxie, recommends you defeat enemies at Resol Forest to gain more field experience."""
```
Comment (DabirSA) [inline Translated/Tasks/Side/Hoxie/ss0030761201.csv]: Having "Northern" here makes the punctuation fall off the end. There's no graceful solution that's perfectly consistent across all four, but this is the least inconsistent, so just leave this one how it is.

File: Translated/UI/ui_mainmode_reb.csv [hunk 4]
```diff
@@ -172,7 +172,7 @@ FighterLimitBreakPhotonCharge#0,"""リミットブレイク終了時に、<br>PP
 FighterLimitBreakInsurance#0,"""During Limit Break, if you would be incapacitated by damage, you survive with 1 HP and become temporarily invincible, but Limit Break ends.<br><yellow>※Main Class only.<c>"""
 KnuckleDodgeHeatRevenge#0,"""When you successfully evade an attack with your Weapon Action, your Normal Attack combo skips to its fifth step.<br><yellow>※Knuckles only.<c>"""
 FighterKnuckleSwayCounterPlus#0,"""Using your Weapon Action after Knuckle Sway Counter activates will generate an additional counterattack."""
-FighterKnuckleNormalAttackPlus#0,"""通常攻撃段数が、最大５段階に増加する。<br><yellow>※ナックル専用<c>"""
+FighterKnuckleNormalAttackPlus#0,"""Increases the number of steps in your Normal Attack combo to 5.<br><yellow>※Knuckles only.<c>"""
```
Comment (DabirSA) [inline Translated/UI/ui_mainmode_reb.csv]: This one is unused, please revert.

**Pull request merged**